modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/management_toolbars.jsp

```diff
@@ -68,6 +68,6 @@ ClaySampleManagementToolbarsDisplayContext managementToolbarsDisplayContext = ne
 <h3>USING DISPLAY CONTEXT</h3>
 
 <clay:management-toolbar
-	displayContext="<%= managementToolbarsDisplayContext %>"
+	managementToolbar="<%= managementToolbarsDisplayContext %>"
```

Comment:
It's a bit weird that this is `managementToolbar**s**DisplayContext` in plural 😂

Reply:
There are more toolbars on the same page in `sample-web`, and the page is named `Management Toolbars`. Makes sense, just barely :slightly_smiling_face: